streamingpro-core/src/main/java/streaming/core/strategy/platform/SparkRuntime.scala

```diff
@@ -106,9 +106,9 @@ class SparkRuntime(_params: JMap[Any, Any]) extends StreamingRuntime with Platfo
     if (MLSQLConf.MLSQL_CLUSTER_PS_ENABLE.readFrom(configReader)) {
       logInfo("PSExecutor configured...")
       conf.set("spark.network.timeout", MLSQLConf.MLSQL_PS_NETWORK_TIMEOUT.readFrom(configReader) + "s")
+      conf.set("spark.plugins", "org.apache.spark.ps.cluster.PSExecutorPlugin")
```

Comment:
Is there any way we can test whether it works?

Reply:
Yes，I will add it in the comment below .